tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -51,5 +220,7 @@ def test_settings_app(self):
         hotspot_page.tap_security_settings()
         self.take_screenshot('internet_sharing-hotspot-security')
         hotspot_page.confirm_security_settings()
-        settings.return_to_prev_menu(internet_settings_page.screen_element)
-        settings.return_to_prev_menu(settings.screen_element)
+        settings.return_to_prev_menu(internet_settings_page.screen_element,
+                                     hotspot_page.screen_element)
+        settings.return_to_prev_menu(settings.screen_element,
+                                     internet_settings_page.screen_element)
```

Comment:
Oh, after running this test, my data connection was enabled.

You might want to add a tearDown function where you disable this again, like here:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py#58